lib/puppet/util/rdoc/code_objects.rb

```diff
@@ -96,14 +96,14 @@ def nodes
   class PuppetClass < ClassModule
     include AddClassModuleComment
 
-    attr_accessor :resource_list, :requires, :childs, :realizes
+    attr_accessor :resource_list, :requires, :children, :realizes
```

Comment:
ping @hlindberg is there a ticket to remove the `doc` puppet application from puppet 6 as part of PUP-7405? If not can you create one?

Reply:
@l0b0 The rdoc stuff is deprecated and we want to remove it. Rather than take the risk of changing it, lets take out the "childs" to "children" change, and let it disappear when we finally remove the code.